src/Symfony/Component/Messenger/Bridge/Redis/Transport/Connection.php

```diff
@@ -157,6 +178,49 @@ private static function validateOptions(array $options): void
         }
     }
 
+    private function claimOldPendingMessages()
+    {
+        try {
+            // This could soon be optimized with https://github.com/antirez/redis/issues/5212 or
+            // https://github.com/antirez/redis/issues/6256
+            $pendingMessages = $this->connection->xpending($this->stream, $this->group, '-', '+', 1);
```

Comment:
@toooni Just to double-check, are we sure that this method always return an array? The `\Redis` API has a lot of undocumented falsy return values in which case we consider that the call failed, see other method calls in the class.

Reply:
Yes. It works with no results, one or multiple results. To be sure I've checked the redis docs again:
> Return value
Array reply, specifically:
The command returns data in different format depending on the way it is called, as previously explained in this page. However the reply is always an array of items.

Reply:
Thank you.